bitcoin/src/psbt/serialize.rs

```diff
@@ -409,21 +409,20 @@ impl Deserialize for TapTree {
         let mut builder = TaprootBuilder::new();
         let mut bytes_iter = bytes.iter();
         while let Some(depth) = bytes_iter.next() {
-            let version = bytes_iter.next().ok_or(Error::ParseFailed("Invalid Taproot Builder"))?;
+            let version = bytes_iter.next().ok_or(Error::TaprootError("Invalid Taproot Builder"))?;
             let (script, consumed) = deserialize_partial::<ScriptBuf>(bytes_iter.as_slice())?;
             if consumed > 0 {
                 bytes_iter.nth(consumed - 1);
             }
-
             let leaf_version = LeafVersion::from_consensus(*version)
-                .map_err(|_| Error::ParseFailed("Leaf Version Error"))?;
+                .map_err(|_| Error::TaprootError("Leaf Version Error"))?;
```

Comment:
Should this one be `InvalidLeafVersion` rather than `TaprootError`?

Reply:
Yes, I'll fix this.